### PR TITLE
Estonia

### PR DESCRIPTION
**Pull request by Vesihiisi**
Base class to process Estonia data.

Preview: https://www.wikidata.org/wiki/Wikidata:WikiProject_WLM/Mapping_tables/ee_(et)/preview

Task: https://phabricator.wikimedia.org/T168361

### CODE REVIEW

File: importer/EeEt.py [hunk 1]
```diff
@@ -5,30 +5,136 @@
 class EeEt(Monument):
 
     def update_labels(self):
```
Comment (lokal-profil) [inline importer/EeEt.py]: Think this can be entirely replaced by `self.set_labels("et", self.nimi)`

Comment (Vesihiisi) [inline importer/EeEt.py]: Done.

**Comment by lokal-profil** [comment]
Would you mind updating https://www.wikidata.org/wiki/Wikidata:WikiProject_WLM/Mapping_tables/ee_(et) with the mapping and the status in the overview table

**Comment by Vesihiisi** [comment]
Re: general comment 1. No.

**Comment by lokal-profil** [comment]
rebased on top of 24c3cfca2ee4f037da58184166e1b8138d3459e0

**Comment by lokal-profil** [comment]
Might be worth taking a look at the changes in Monument and util to see if they are still desired after the rebase.

**Comment by Vesihiisi** [comment]
I've updated the file wrt the recent changes in Monument/utils and also added register url as quantifier to heritage id.

File: importer/EeEt.py [hunk 2]
```diff
@@ -51,10 +47,11 @@ def update_heritage(self):
             except IndexError:
                 return
 
-    def set_no(self):
+    def set_heritage_id(self):
         """Set Estonian cultural monument ID (P2948)."""
+        qual_url = {"described_at_url": self.registrant_url}
```
[the remark this answers is not in the document]
Comment (Vesihiisi) [reply importer/EeEt.py]: Removed it.

File: importer/EeEt.py [hunk 3]
```diff
@@ -49,9 +49,8 @@ def update_heritage(self):
 
     def set_heritage_id(self):
         """Set Estonian cultural monument ID (P2948)."""
-        qual_url = {"described_at_url": self.registrant_url}
         register_no = str(self.registrant_url.split("=")[-1])
```
Comment (lokal-profil) [inline importer/EeEt.py]: think you can just use `self.number` here

Comment (Vesihiisi) [reply importer/EeEt.py]: Done.

**Pull request merged**